src/event_object_movement.c

```diff
@@ -7900,6 +9420,40 @@ static void DoTracksGroundEffect_Footprints(struct ObjectEvent *objEvent, struct
     FieldEffectStart(sandFootprints_FieldEffectData[isDeepSand]);
 }
 
+static void DoTracksGroundEffect_FootprintsB(struct ObjectEvent *objEvent, struct Sprite *sprite, bool8 isDeepSand)
+{
+    // First half-word is a Field Effect script id. (gFieldEffectScriptPointers)
+    u16 otherFootprintsA_FieldEffectData[2] = {
```

Comment:
As before, shouldn't this be a static outside of this function? Same for the ones below

Reply:
The object event template I think should stay; it's not static, most of its data depends on values at runtime